request.proto

```diff
@@ -165,7 +165,7 @@ message NearestStopPointsRequest {
 
 message GraphicalIsochroneRequest {
     optional JourneysRequest journeys_request = 1;
-    required int32 min_duration               = 2;
+    repeated int32 duration                   = 4;
```

Comment:
Every `param[]` is singular except `forbidden_uris[]`. That's why `duration[]` is singular.